### ISSUE TITLE
Emojione's name clashes with default Apple Color Emoji when installed

### ISSUE THREAD

**Issue by googlercolin** [Ranks/emojione#406]
Is there a different name for Emojione after installation such that it does not clash with the default Apple Color Emoji so that both fonts can co-exist instead of overriding each other?

**Comment by Crissov** [comment]
It’s a feature, not a bug of `emojione-apple.ttf`. It’s explained in more detail at [Mac Issues](http://www.macissues.com/2014/11/21/how-to-change-the-default-system-font-in-mac-os-x/), but remember that the relevant PS name is `AppleColorEmoji` for emoji fonts. [`EmojiOneColor-SVGinOT-OSX.ttf`](https://github.com/eosrei/emojione-color-font/#install-on-os-x) employs the same hack.

You can edit the file manually as described in the article linked to above, but I think the font rendering layer of OS X / macOS will override emojis from other fonts that are also available in the active copy of `AppleColorEmoji`.

**Comment by googlercolin** [comment]
Hmm so that means all Emoji fonts will be called AppleColorEmoji?

**Comment by Crissov** [comment]
Yes, at least if they’re supposed to become the default emoji font (or Apple changes how font selection works).

**Comment by googlercolin** [comment]
Oh but if I don’t want Emojione to be my default Emoji font, but a secondary one, what should I do?

**Comment by Crissov** [comment]
You should be able to use any font editor to change the Postscript name in the file, doesn’t have to be [TTFEdit](https://sourceforge.net/projects/ttfedit/). If you are comfortable editing XML, [ttx](https://github.com/fonttools/fonttools) should do. (It’s also part of [AFDKO](http://www.adobe.com/devnet/opentype/afdko/topic_overview.html).)

**Comment by googlercolin** [comment]
Hi Crissov, I'm not very sure what you meant by "A version like that could be automatically build and released, of course, e.g. as emojione-sbix.ttf or even as a cross-platform emojione-png.ttf." 😅

**Comment by googlercolin** [comment]
@Crissov When I tried changing the PostScript name in TTFEdit, and importing it into Font Book, this error was encountered. May I seek your assistance in this?
<img width="1440" alt="screen shot 2017-01-14 at 9 51 12 am" src="https://cloud.githubusercontent.com/assets/18029177/21951287/4fe3f8c2-da3f-11e6-83ea-81ae41ea5b7f.png">

**Comment by caseyahenson** [comment]
@miguelsousa Can you help advise on this? Recommendations for a workaround so that EmojiOne can be referenced as an installed font but not replace all system emoji? Thanks in advance.
